src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -67,8 +89,70 @@ def execute_mdtest(self):
         # Create a pool
         self.pool.create()
 
-        # Run Mdtest
+    def create_cont(self):
+        """Create a TestContainer object to be used to create container."""
+        # TO-DO: Enable container using TestContainer object,
+        # once DAOS-3355 is resolved.
+        # Get Container params
+        #self.container = TestContainer(self.pool)
+        #self.container.get_params(self)
+
+        # create container
+        # self.container.create()
+        env = Dfuse(self.hostlist_clients, self.tmp).get_default_env()
+        # command to create container of posix type
+        cmd = env + "daos cont create --pool={} --svc={} --type=POSIX".format(
+            self.mdtest_cmd.dfs_pool_uuid.value, self.mdtest_cmd.dfs_svcl.value)
+        try:
+            container = subprocess.Popen(cmd, stdout=subprocess.PIPE,
+                                         shell=True)
+            (output, err) = container.communicate()
+            self.log.info("Container created with UUID %s", output.split()[3])
+
+        except subprocess.CalledProcessError as err:
+            self.fail("Container create failed:{}".format(err))
+
+        return output.split()[3]
+
+    def start_dfuse(self):
+        """Create a DfuseCommand object to start dfuse."""
+        # Get Dfuse params
+        self.dfuse = Dfuse(self.hostlist_clients, self.tmp, self.basepath)
+        self.dfuse.get_params(self)
+
+        # update dfuse params
+        self.dfuse.set_dfuse_params(self.pool)
+        self.dfuse.set_dfuse_cont_param(self.create_cont())
+
+        try:
+            # start dfuse
+            self.dfuse.run()
+        except CommandFailure as error:
+            self.log.error("Dfuse command %s failed on hosts %s",
+                           str(self.dfuse), str(NodeSet(self.dfuse.hosts)),
+                           exc_info=error)
+            self.fail("Test was expected to pass but it failed.\n")
+
+
+    def execute_mdtest(self):
```

Comment:
Assuming this is the actual test it starts dfuse it's then stopped in the tearDown() outside of the test, so it would be better to stop dfuse from this function as well so shutdown errors could be reported properly as well.

Reply:
Sure will do.